SmRtAPI/DemoApp/Program.cs

```diff
@@ -51,10 +52,11 @@ public static void Main(string[] args)
                 {
                     /*
                      * The API constructor is passed the websockets URL, callbacks for the messages it might receive,
-                     * the language to transcribe (as a .NET CultureInfo object) and stream to read data from.
+                     * the language to transcribe and stream to read data from.
                      */
                     var config = new SmRtApiConfig(language)
                     {
+                        AuthToken = "99m94ktkOLVrnwKUqG33LB1YrLqGE2yZ",
```

Comment:
I'd put a comment to re-assure the reader that this isn't a valid token (or put `<invalid_token>` as a placeholder or something)